css/docs.css

```diff
@@ -0,0 +1,155 @@
+/* Global Variables */
+:root {
+  /* color */
+  --header-bg: #1F2937;
+  --light-grey: #F1F3F5;
+  --pure-white: #FFFFFF;
+  --white: #e8e8fd;
+  --black: #18181B;
+  --dark: #20232A;
+  --grey: #E5E7EB;
+  /* spacing */
+  --side-nav-width: 15rem;
+  --header-height: 4.37rem;
+}
+
+/* Default Styling */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.highlight {
+  background-color: var(--grey);
+  font-family: monospace;
+}
+
+pre {
+  padding: 1rem;
+}
+
+code {
+  border-radius: 0.5rem;
+}
+
```

Comment:
pre and code tags are not supported in HTML5

Reply:
I checked and it turns out that they are supported.
[HERE](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-code-element.html) is the W3C link regarding this, please check it out.